xen.spec.in

```diff
@@ -633,20 +535,30 @@ rm %{buildroot}%{_sysconfdir}/rc.d/init.d/xen-watchdog
 rm %{buildroot}%{_sysconfdir}/rc.d/init.d/xencommons
 rm %{buildroot}%{_sysconfdir}/rc.d/init.d/xendomains
 rm %{buildroot}%{_sysconfdir}/rc.d/init.d/xendriverdomain
+# BEGIN QUBES SPECIFIC PART
 rm %{buildroot}%{_sysconfdir}/sysconfig/xendomains
-
 cp %{SOURCE32} %{buildroot}/usr/lib/modules-load.d/xen.conf
 
-# Qubes specific - get rid of standard domain starting scripts
+# get rid of standard domain starting scripts
 rm %{buildroot}%{_unitdir}/xen-qemu-dom0-disk-backend.service
 rm %{buildroot}%{_unitdir}/xendomains.service
+# END QUBES SPECIFIC PART
 
 ############ create dirs in /var ############
 
 mkdir -p %{buildroot}%{_localstatedir}/lib/xen/images
 mkdir -p %{buildroot}%{_localstatedir}/log/xen/console
 
+############ create symlink for x86_64 for compatibility with 4.4 ############
+
+%if "%{_libdir}" != "/usr/lib"
+ln -s %{_libexecdir}/%{name} %{buildroot}/%{_libdir}/%{name}
+%endif
+
+# BEGIN QUBES SPECIFIC PART
+# don't create symlink to qemu-system-i386
```

Comment:
This comment is for something else, no?

Reply:
No it's me who made this one with respect to https://src.fedoraproject.org/rpms/xen/blob/f25/f/xen.spec#_590